app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -134,6 +155,17 @@ angular.module("openshiftConsole")
 
           scope.$watchGroup(['claim.amount', 'claim.unit'], validateLimitRange);
         });
+
+        DataService.list('resourcequotas', { namespace: scope.projectName }, function(quotaData) {
+          var quotas = quotaData.by('metadata.name');
+          if (_.isEmpty(quotas)) {
+            return;
+          }
+
+          scope.storageQuota = QuotaService.getStorageQuotas(quotas);
+          console.log("Storage Quotas",scope.storageQuota);
```

Comment:
if you are wanting to keep this debug logging it needs to use the Logger service instead, we dont check in console.log stmts